src/services/api-service.ts

```diff
@@ -21,9 +21,9 @@ export function getGradeList(): Promise<IGrade[]> {
     .catch(e => console.log('Could not get grade list'));
 }
 
-export function getSubjectList(): Promise<ISubject[]> {
+export function getSubjectList(mestring: string): Promise<ISubject[]> {
   return api
-    .get('subjects')
+    .get('subjects' + mestring)
     .then(res => res.data)
     .catch(e => console.log('Could not get subject list'));
```

Comment:
Bruk gjerne console.error i det minste eller faktisk ha noe feilhåndtering som ikke bare er logging. F.eks en notification av et slag. Det finnes sikkert noe bibliotek for dette. Det hadde vært greit å ha for flere andre ting også